eos-updater-flatpak-installer/eos-updater-flatpak-installer.service.in

```diff
@@ -1,12 +1,12 @@
 [Unit]
 Description=Endless OS Post-Boot Flatpak Installer
-Wants=local-fs.target
-After=local-fs.target
+Wants=network.target
+After=network.target
 ConditionKernelCommandLine=!eos-updater-disable
+DefaultDependencies=no
 
 # Only on updates
 Before=multi-user.target systemd-update-done.service
-ConditionNeedsUpdate=/var
```

Comment:
If you don't have `ConditionNeedsUpdate`, then there's no reason to have `Before=systemd-update-done.service` since it will run regardless of if an update was found.

If you don't need this to run only update, then you can remove `DefaultDependencies=no` and just make this a normal service. If you do want this to run only on update, then you need to put back a `ConditionNeedsUpdate` setting and resolve the ordering cycle.

Reply:
@cosimoc and I talked about this and we resolved on keeping `ConditionNeedsUpdate` - I had misunderstood how it worked we figured out a way to test it.